nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -191,56 +191,6 @@ in
     };
   };
 
-  # TODO: remove this
-  systemd.services.fetch-ssh-keys =
-    { description = "Fetch host keys and authorized_keys for root user";
```

Comment:
The google-accounts-daemon also scans for the SSH keys, but creates individual accounts per SSH key owner (with passwordless sudo). The root account isn't directly accessible through SSH anymore.